### PR TITLE
Add ternaryline option

### PR DESCRIPTION
**Pull request by szeck87**
Add option `ternaryline` to options.ts.

### CODE REVIEW

**Comment by szeck87** [comment]
@Glavin001 can you please review/merge/publish?

File: src/options.ts [hunk 1]
```diff
@@ -351,6 +351,11 @@ export const Options: OptionsRegistry = {
     ],
     type: "string"
   },
+  ternaryline: {
```
[the remark this answers is not in the document]
Comment (Glavin001) [reply src/options.ts]: Let's go with `multiline_ternary`: https://eslint.org/docs/rules/multiline-ternary 

And let’s make it enum, `always, always-multiline, never`.
We should provide the most configuration options as possible, so boolean does not cut it in this case.

File: src/options.ts [hunk 2]
```diff
@@ -253,6 +253,16 @@ export const Options: OptionsRegistry = {
     description: "Number of line-breaks to be preserved in one chunk",
     type: "integer"
   },
+  multiline_ternary: {
+    default: false,
```
Comment (Glavin001) [inline src/options.ts]: `default` needs to be updated. `always` is what ESLint has as default.

File: src/options.ts [hunk 3]
```diff
@@ -253,6 +253,16 @@ export const Options: OptionsRegistry = {
     description: "Number of line-breaks to be preserved in one chunk",
     type: "integer"
   },
+  multiline_ternary: {
+    default: "always",
+    description: "Keeps ? and : operators of ternary statements on the same line of code",
```
Comment (Glavin001) [inline src/options.ts]: I think this `description` needs to be the opposite.

From https://eslint.org/docs/rules/multiline-ternary :
`always` => `enforces newlines between the operands of a ternary expression.`

File: src/options.ts [hunk 4]
```diff
@@ -253,6 +253,16 @@ export const Options: OptionsRegistry = {
     description: "Number of line-breaks to be preserved in one chunk",
     type: "integer"
   },
+  multiline_ternary: {
+    default: "always",
+    description: "Enforces new lines between the operands of a ternary expression",
+     enum: [
```
Comment (Glavin001) [inline src/options.ts]: Is it just me or did the indentation change and `enum` property is messed up?

**Pull request merged**